File: PromizzApp.Client/src/app/shared/models/user.model.ts
export class UserModel {
    public Id: number = 0;
    public Firstname: string = '';
    public Lastname: string = '';
    public UserName: string = '';
    public Fullname: string = '';
    public Initials: string = '';
    public Email: string = '';
    public Avatar: string = '';
    public Bio: string = '';
    public Active: boolean = true;
    public Password: string = '';
    public Token: string = '';

    public constructor(init?: Partial<UserModel>) {
        Object.assign(this, init);
    }
}

File: Models/PromizzApp.Models/UserModel.cs

namespace PromizzApp.Models
{
    public class UserModel
    {
        public int Id { get; set; }

        public string Firstname { get; set; }

        public string Lastname { get; set; }

        public string Username { get; set; }

        public string Fullname
        {
            get => Firstname + " " + Lastname;
        }

        public string Initials
        {
            get => Config.Utils.GetFirstLettersFromStrings(Firstname, Lastname).ToUpper();
        }
    
        public string Email { get; set; }

        public string Bio { get; set; }

        public string Avatar { get; set; }
    }
}
